packages/registry/docs/index.md

```diff
@@ -32,26 +25,37 @@ Add-on packages are just CommonJS/ESM packages.
 Their main purpose is encapsulate logic, configuration and customizations in a reusable way.
 The only requirement is that their primary entry point (`main` key of their `package.json`) points to a module that exports a default function, which acts as a default configuration loader for that package.
 
-Add-ons are applied in the order they are declared in the `addons` key of `package.json` or programatically via the configuration file (`volto.config.js` or `registry.config.js`).
+Add-ons are applied in the order they are declared in the `addons` key of `package.json` or programatically via a provided configuration file.
 Add-ons can override configuration coming from other add-ons, providing a hierarchy of configuration stacks.
 
 An add-on can be published in an npm registry, just as any other package.
 However, add-ons are meant to not be transpiled, but built along with your app code.
 They can be released as "source" packages or used directly in your app as local code.
 
+Add-ons can define shadowed components.
+"Component shadowing" is a technique for overriding modules of other packages on build time.
+This technique builds upon the `resolve.aliases` facilities of bundlers, so modules can be replaced when the app is being built.
+
+Add-ons can be chained, so each one can configure the app in some way, but if needed, the next add-on can override and extend the previous configuration set by other add-ons. Thus, the order in which is add-on is registered matters.
+
 % QUESTION: Should this go to "HowTo"s?
```

Comment:
Yes. I'm not sure whether a `toctree` or `seealso` directive is more appropriate. We'll need to look at it again, after the dust settles.